src/imgui_tools.cpp

```diff
@@ -19,8 +19,11 @@
 #include "d3d12/d3d12_pipeline_registry.hpp"
 #include "imgui/ImGuizmo.h"
 
+
 namespace wr::imgui::internal
 {
+	struct RTAOData;
```

Comment:
unused?

Reply:
correct, was unused in the current implementation, to make things worse, what it was initially implemented is removed due to feedback on PR :feelsBadMan: